src/index.css

```diff
@@ -1,76 +1,353 @@
 body {
-  background-color: rgb(250, 250, 250);
+  margin: 0;
 }
 
 * {
+  margin: 0;
   padding: 0;
+}
+
+#root {
```

Comment:
It's important to add all elements of the design. Please add the user icon to your project 🚀

![image](https://user-images.githubusercontent.com/15807343/185691662-4e2a27a2-246e-4d59-bb41-a2f29fea2673.png)